app/seeds/03_missions.js

```diff
@@ -8,19 +8,22 @@ exports.seed = function(knex, Promise) {
         name: 'Period 1',
         casefile_id: 1,
         user_id: 1,
-        url: 'http://localhost:3000/mstestteacher/1'
+        url: 'http://localhost:3000/mstestteacher/1',
```

Comment:
these work for now, but we'll have to think about how we'll want to update this to a relative path so it would work in production too.